src/js/common/components/LoadingIndicator.jsx

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
 
-export default function LoadingIndicator({ message }) {
-  return (
-    <div className="loading-indicator-container">
-      <div className="loading-indicator"></div>
-      {message}
-    </div>
-  );
+export default class LoadingIndicator extends React.Component {
+  componentDidMount() {
+    if (this.props.setFocus) {
+      this.spinnerDiv.focus();
+    }
+  }
+  render() {
+    const { message, screenReaderMessage } = this.props;
+
+    return (
+      <div className="loading-indicator-container">
+        <div
+            ref={(div) => { this.spinnerDiv = div; }}
+            className="loading-indicator"
+            role="progressbar"
+            aria-valuetext={message || screenReaderMessage}
+            tabIndex="0"></div>
+        {message}
+      </div>
+    );
+  }
 }
+
+LoadingIndicator.propTypes = {
+  message: React.PropTypes.string,
```

Comment:
@jbalboni -- Can the `message` and the `screenReaderMessage` say the same thing? It is important to keep the experience consistent for sighted and non-sighted users. and the we can may the `message` `propType` required.